### DISCUSSION TITLE
Network filter doesn't get applied

### DISCUSSION THREAD

**Discussion by uBlock-user** [uBlockOrigin/uBlock-issues#2691] [General] [answered]
- Add `||vidstream.pro/asset/com/$script,3p` 

- Browse to `https://fmovies.to/movie/fast-x-0mllk/1-1` and using the logger, observe the filter above doesn't get applied by uBO ???

Did something change/break ?

@gorhill 

Edit: `||vidstream.pro/asset/com/$script,1p` works. So partyness logic changed ?

v1.49.3rc16

**Comment by gorhill** [comment]
The context of the request `https://vidstream.pro/asset/com/...` is `https://vidstream.pro/embed/51J0DVM63PQ8?...` (an iframe), so all work as expected.

**Reply by uBlock-user** [reply]
But the iframe is 3rd party to the domain `fmovies.to`

**Reply by gorhill** [reply, marked as answer]
The context of a network request is always the context causing that network request to happen. This has **always** be the case, nothing changed. The only time uBO care about the root context is with dynamic filtering and per-site switches.

**Reply by MasterKia** [reply]
https://github.com/uBlockOrigin/uAssets/discussions/14705
